### PR TITLE
Campaign page

### PR DESCRIPTION
**Pull request by sarunast**
Campaign page. 
The actual URL is set to `/kampagne` not sure if this is good. Maybe `/campaign` would be understandable as German version.

By the way @erikjalevik I added the link to campaign the page in the footer.

screen shot 2018-03-06 at 17 23 26

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 3182debd54eaf73ff29b6aa6ba16ba482db9c43f

https://deploy-preview-201--masifunde.netlify.com

**Comment by erikjalevik** [comment]
I don't think '/kampagne' is the best choice, I believe Masifunde call it '/spendenaktion'.

**Comment by erikjalevik** [comment]
The campaigns should not have a footer link, as they are not always available. Seems I forgot to create an implementation ticket for the work of integrating the campaign links, see here:

https://trello.com/c/a4FM6irh/356-integrate-current-campaign-into-site-hierarchy

**Comment by erikjalevik** [comment]
How come there's a "your impact with €15" only appearing for that amount?

File: api/campaign.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { fetchSingleEntry } from './contentfulService'
+import { unwrapFields, unwrapImages, unwrapTeamMember } from './common'
+
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (erikjalevik) [inline api/campaign.js]: Why not disable the linter rule instead?

File: pages/campaign/index.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import DonationForm from '../../components/DonationForm'
+import imagePropTypes from '../../propTypes/image'
+import teamMemmberPropTypes from '../../propTypes/teamMember'
+
+const StyledRoundedImage = RoundedImage.extend`
+  margin-bottom: 1rem;
+  display: block;
+`
+
+const TeamMemberHeadline = styled.div`
```
Comment (erikjalevik) [inline pages/campaign/index.js]: We don't have a heading style for this already? Is it a unique heading?

**Comment by sarunast** [comment]
I will change the things that you mentioned in the comments. The 15 EUR description is visible because it is the only that is defined if you go to Contentful or spenden page you will see that they don't have a description.

File: pages/campaign/index.js (hunk 2 above)
Comment (sarunast) [reply pages/campaign/index.js]: I think it is. If you think it is used anywhere I can switch it.

File: api/campaign.js (hunk 1 above)
Comment (sarunast) [reply api/campaign.js]: I will do it.

**Pull request merged**